app/javascript/shared/components/MarkdownEditor.res

```diff
@@ -291,27 +282,18 @@ let controls = (disabled, value, state, send, onChange) => {
       <div />
     | Windowed(#Editor)
     | Fullscreen(#Editor | #Split) =>
-      <div role="toolbar" className="bg-white border border-gray-400 rounded-t border-b-0">
-        <button
-          disabled
-          ariaLabel={t("control_label_bold")}
-          title={t("control_label_bold")}
-          className=buttonClasses
-          onClick={_ => curriedModifyPhrase(Bold)}>
```

Comment:
@indicozy - This file has lot of changes unrelated to this PR. Could you please stash or move them to a new PR?